src/org/etools/j1939_84/utils/StringUtils.java

```diff
@@ -76,4 +76,18 @@ private static String stripTrailingNulls(String input) {
         }
     }
 
+    public static String capitalizeString(String string) {
```

Comment:
A unit test for this would be nice.

Reply:
It. has annotation and has to go with the very large check in that is coming. Still working on updates from Joe from the feedback he gave on Friday.